.github/workflows/e2e.yaml

```
@@ -65,14 +65,18 @@ jobs:
         openshift_password: ${{ secrets.OPENSHIFT_PASSWORD }}
         insecure_skip_tls_verify: true
         namespace: default
+    - run: |
+          for i in `oc get specialresources.sro.openshift.io -o jsonpath='{.items[*].metadata.name}'`;
+            do oc patch specialresources.sro.openshift.io $i -p '{"metadata":{"finalizers":[]}}' --type=merge;
+            oc delete specialresources.sro.openshift.io $i --ignore-not-found
+          done
     - run: make deploy
       env:
-        TAG: master
+        TAG: pr-${{ steps.short-sha.outputs.sha }}
+    - run: make test-e2e
+      continue-on-error: true
+    - run: oc logs deployment/special-resource-controller-manager -n openshift-special-resource-operator -c manager
       continue-on-error: true
```

I think we could use `always()` here to make sure that we always get the logs, no matter what any earlier command returned.
```suggestion
      if: ${{ always() }}
```

---

Not sure about this one. If `make deploy` fails I dont want to try getting logs because there are no pods there.

---

Linked to the thread below - resolving this one.